lib/api.rb

```diff
@@ -69,13 +116,23 @@ def validation_error(msg)
 
   # we're always going to reply with json
   content_type :json
+  response.headers['Access-Control-Allow-Origin'] = '*'
```

Comment:
if the web-app is hosted under the same domain as the backend, then this is also not needed.

Reply:
You’re right. But it need a reverse proxy in front (that’s what i did).